dashboard/src/modules/components/ProfileComponent/KeyListTable.jsx

```diff
@@ -0,0 +1,75 @@
+import { Button, ClipboardCopy } from "@patternfly/react-core";
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { useDispatch, useSelector } from "react-redux";
+
+import React from "react";
+import { TrashIcon } from "@patternfly/react-icons";
+import { formatDateTime } from "utils/dateFunctions";
+import { triggerDeleteAPIKey } from "actions/keyManagementActions";
+
+const KeyListTable = () => {
+  const dispatch = useDispatch();
+  const keyList = useSelector((state) => state.keyManagement.keyList);
+  const columnNames = {
+    label: "Label",
+    created: "Created Date & Time",
+    key: "API key",
+  };
+
+  const deleteKey = (id) => {
+    dispatch(triggerDeleteAPIKey(id));
+  };
+
+  return (
+    <TableComposable aria-label="key list table" isStriped>
+      <Thead>
+        <Tr>
+          <Th width={10}>{columnNames.label}</Th>
+          <Th width={20}>{columnNames.created}</Th>
+          <Th width={20}>{columnNames.key}</Th>
+          <Th width={5}></Th>
+        </Tr>
+      </Thead>
+      <Tbody className="keylist-table-body">
+        {keyList.map((item) => {
+          return (
+            <Tr key={item.key}>
```

Comment:
I think it's better to use a simple expression rather than a code block containing just a `return` statement.  I.e.,
```javascript
        {keyList.map((item) => (
          <Tr key={item.key}>
```